spec/spec_helper.rb

```diff
@@ -24,12 +22,24 @@ class SecondUser < Minidoc
   attribute :age, Integer
 end
 
+class ValidationsUser < User
```

Comment:
What's stopping us from declaring this locally to the test scope, similar to how it used to be?